.github/actions/spelling/patterns.txt

```diff
@@ -224,3 +224,6 @@ user:P@ssw0rd
 
 # Ignore base64 encoded values in Prometheus Pushgateway URL paths
 /.+@base64/.+
+
+# changelog.d fragment authors line
```

Comment:
cc @neuronull

Reply:
Thank you for checking!

@klondikedragon there's a few doc suggestions, after that this should be ready to merge.

Reply:
thanks @maycmlee - @pront should be ready for another check